contracts/andromeda_gumball/src/contract.rs

```diff
@@ -0,0 +1,881 @@
+    let state = STATE.load(deps.storage)?;
+    Ok(StateResponse { state })
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use common::ado_base::recipient::Recipient;
+    use common::mission::AndrAddress;
+    use cosmwasm_std::testing::{mock_dependencies, mock_env, mock_info};
+    use cosmwasm_std::{coin, from_binary, to_binary};
+
+    fn mint(deps: DepsMut, token_id: impl Into<String>) -> Result<Response, ContractError> {
+        let msg = ExecuteMsg::Mint(Box::new(MintMsg {
+            token_id: token_id.into(),
+            owner: mock_env().contract.address.to_string(),
+            token_uri: None,
+            extension: TokenExtension {
+                name: "name".to_string(),
+                publisher: "publisher".to_string(),
+                description: None,
+                transfer_agreement: None,
+                metadata: None,
+                archived: false,
+                pricing: None,
+            },
+        }));
+        println!("check 4");
+
+        execute(deps, mock_env(), mock_info("owner", &[]), msg)
+    }
+    #[test]
+    fn swap_rem() {
+        let mut v: Vec<u8> = vec![
+            12, 4, 1, 84, 179, 120, 223, 73, 87, 30, 84, 218, 109, 137, 198, 191, 55, 238, 215,
+            158, 129, 164, 35, 40, 96, 219, 56, 72, 174, 106, 132, 143,
+        ];
+
+        let removed_element = v.swap_remove(1);
+        assert_eq!(removed_element, 4);
+    }
+    #[test]
+    fn binary() {
+        let v: Vec<u8> = vec![
+            12, 4, 1, 84, 179, 120, 223, 73, 87, 30, 84, 218, 109, 137, 198, 191, 55, 238, 215,
+            158, 129, 164, 35, 40, 96, 219, 56, 72, 174, 106, 132, 143,
+        ];
+
+        let bin = to_binary(&v).unwrap();
+        println!("{:?}", bin);
+
+        let random_hex: Vec<u8> = from_binary(&bin).unwrap();
+        println!("{:?}", random_hex);
+    }
+    #[test]
+    fn bytes() {
+        let _v: Vec<u8> = vec![
+            12, 4, 1, 84, 179, 120, 223, 73, 87, 30, 84, 218, 109, 137, 198, 191, 55, 238, 215,
+            158, 129, 164, 35, 40, 96, 219, 56, 72, 174, 106, 132, 143,
+        ];
+        let n = "2b51af9c2bc12b262e2fc955bcb9fab4c89375efee6210385c40f59948e539d6".to_string();
+        let tbin = Binary::from_base64(&n).unwrap();
+        println!("from base64: {:?}", tbin);
+        let bin = to_binary(&n).unwrap();
+        println!(" to binary: {:?}", bin);
+        let trandom = Binary::to_base64(&tbin);
+        println!(" to_base64{:?}", trandom);
+        let random_hex: String = from_binary(&bin).unwrap();
+        println!(" from_binary{:?}", random_hex);
+
+        let vec = trandom.into_bytes();
+        println!("trandom into bytes{:?}", vec);
+        let ran_vec: Vec<u64> = vec.iter().map(|x| *x as u64).collect();
+        let random_number: u64 = ran_vec.iter().sum();
+        println!("{:?}", random_number);
+        let index = random_number % 3;
+        println!("{:?}", index);
+    }
```

Comment:
Since these tests are not testing logic that is unique to us they should be removed.